RNLiveMarkdown.podspec

```diff
@@ -5,8 +5,23 @@ react_native_json = JSON.parse(File.read(File.join(react_native_node_modules_dir
 react_native_minor_version = react_native_json['version'].split('.')[1].to_i
 
 pods_root = Pod::Config.instance.project_pods_root
-react_native_reanimated_node_modules_dir = ENV['REACT_NATIVE_REANIMATED_NODE_MODULES_DIR'] || File.dirname(`cd "#{Pod::Config.instance.installation_root.to_s}" && node --print "require.resolve('react-native-reanimated/package.json')"`)
-react_native_reanimated_node_modules_dir_from_pods_root = Pathname.new(react_native_reanimated_node_modules_dir).relative_path_from(pods_root).to_s
+
+is_package_installed = ->(name) do
+  package_path = File.join(react_native_node_modules_dir, name)
+  File.directory?(package_path)
```

Comment:
`react-native-worklets` can technically be installed as a link and in that case `node_modules/react-native-worklets` will be a symlink instead of a directory. Let's find a better way to detect if worklets are installed or not